include/types/wlr_output_layout.h

```diff
@@ -0,0 +1,26 @@
+#ifndef TYPES_WLR_OUTPUT_LAYOUT_H
+#define TYPES_WLR_OUTPUT_LAYOUT_H
+
+#include <wlr/types/wlr_output_layout.h>
+
+struct wlr_output_layout_output {
+	struct wlr_output_layout *layout;
+	struct wlr_output *output;
+	int x, y;
+	struct wl_list link;
+	bool auto_configured;
+
+	struct {
+		struct wl_signal destroy;
+	} events;
+
+	struct wl_listener mode;
+	struct wl_listener scale;
+	struct wl_listener transform;
+	struct wl_listener output_destroy;
+};
+
+struct wlr_output_layout_output *wlr_output_layout_get(
```

Comment:
This function will be exported because it has the `wlr_` prefix. If it's in a private header, it shouldn't.